src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -3264,6 +3265,25 @@ public InputStream getGenericThumbnailInputStream(String fallback) throws IOExce
 			} catch (Exception e) {}
 		}
 
+		if (media != null) {
+			BufferedImage bi;
+			if (media.isAudio()) {
+				bi = PMS.getGenericAudioIcon();
+			} else if (media.isImage()) {
+				bi = PMS.getGenericImageIcon();
+			} else if (media.isVideo()) {
+				bi = PMS.getGenericVideoIcon();
+			} else {
+				bi = PMS.getGenericUnknownIcon();
+			}
+
+			is = ImagesUtil.addFormatLabelToImage(bi, media.getContainer(), defaultRenderer != null && defaultRenderer.isForceJPGThumbnails() ? "jpeg" : "png");
+			if (is != null) {
+				LOGGER.trace("Created default thumbnail/icon for the media: {}", getName());
```

Comment:
Doesn't work as expected as this message is never displayed, whatever is the type of media.